Sources/HaCWebsiteLib/ViewModels/LandingPage/EventFeature.swift

```diff
@@ -56,6 +60,11 @@ struct EventFeature: LandingFeature {
     return paddedEventPeriod.contains(currentDate)
   }
 
+  // Whether the event is on the same day
+  var isToday: Bool {
+    return NSCalendar.current.isDateInToday(eventPeriod.start)
```

Comment:
Neat! 😊

Reply:
What timezone is this in?

Reply:
@Pinpickle This is a good point, as right now it depends on which server this is running on. I updated it so that the calendar has the GMT timezone.